src/Microsoft.ML.PCA/PcaTransformer.cs

```diff
@@ -36,7 +36,7 @@ namespace Microsoft.ML.Transforms.Projections
     /// <include file='doc.xml' path='doc/members/member[@name="PCA"]/*' />
     public sealed class PcaTransformer : OneToOneTransformerBase
```

Comment:
>PcaTransformer [](start = 24, length = 14)

cc @ganik who is doing same thing in his [https://github.com/dotnet/machinelearning/pull/2374](https://github.com/dotnet/machinelearning/pull/2374) PR.